### DISCUSSION TITLE
ArchR multiome pipeline: do we need calculate gene score?

### DISCUSSION THREAD

**Discussion by Jiahuan90** [GreenleafLab/ArchR#1458] [Questions / Documentation] [answered]
Hi rcorces,

I have a question: I have a multiome scATAC-seq + scRNA-seq data. After addCombinedDims() + addUMAP() + addClusters(), my understanding is that I don't need to calculate GeneScore by addGeneScoreMatrix() and addGeneIntegrationMatrix() anymore. 

I could use addCellColData() to import seurat object@meta.data to ArchR project and directly do callPeaks? 
Is my understanding correct?

Thank you for your help~

**Comment by rcorces** [comment, marked as answer]
sounds correct. You're welcome to calculate gene scores (for ex. to compare to the actual gene expression) but its clearly less relevant to analysis once you have ground truth gene expression

**Reply by Jiahuan90** [reply]
Thank your for your kindly help~~

**Reply by Jiahuan90** [reply]
By the way, do I need to do Pseudo-bulk Replicates in my multiome data?

**Reply by mitchTwoTimes** [reply]
Hi, I'd like to piggy back on something @rcorces said here as I'm also looking at multiome data. Isn't gene score just as relevant for a multiome dataset because it's archR's proxy for chromatin accessibility..? If not, how do you recommend quantification of chromatin accessibility for multiome? 

One example of this is seen here in this [publication](https://jasn.asnjournals.org/content/33/7/1308.full) where they use 'chromatin accessibility' and 'gene score' interchangeably.

**Reply by rcorces** [reply]
My apologies if my response wasnt clear. I think if you are trying to talk about gene expression and you have multiome data, then it doesnt make sense to look at gene score because it is an approximation and you already have the ground truth gene expression.

Gene score could of course still be used as a summary value to describe chromatin accessibility around a gene. Though if it were me I would try to focus my chromatin accessibility analyses on regulatory elements rather than using gene scores which are more abstract.
